test/comparison-tests/create-and-execute-test.js

```diff
@@ -371,6 +371,8 @@ function getNormalisedFileContent(file, location, test) {
                 .replace(/[\d]+[.][\d]* kB/g, ' A-NUMBER-OF kB')
                 // We also don't want a difference in the number of bytes to fail the build
                 .replace(/ \d+ bytes /g, ' A-NUMBER-OF bytes ')
+                // Sometimes "[built]" is written to output, and sometimes not. This should not fail the build
```

Comment:
This also happens sometimes in nightly builds: [#1473.20](https://travis-ci.org/TypeStrong/ts-loader/jobs/220300987)